src/server/application/ContractProxy.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { expect } from 'chai'
+import * as http from 'http'
+import { listen, close } from '../lib/http'
+import { InMemoryRepository } from '../infrastructure/repositories'
```

Comment:
```suggestion
import { InMemoryRepository } from '../infrastructure'
```

Reply:
These infrastructure imports are getting a bit messy. I think it makes sense to export them from an index.ts file